cli/src/main/java/keywhiz/cli/commands/UpdateAction.java

```diff
@@ -59,6 +61,19 @@ public UpdateAction(UpdateActionConfig config, KeywhizClient client,
       content = readSecretContent();
     }
     partialUpdateSecret(secretName, content, config);
+
+    // If it appears that content was piped in but --content was not specified, print a warning
+    if (!config.contentProvided) {
+      try {
+        InputStreamReader reader = new InputStreamReader(stream, "UTF-8");
+        if (reader.ready()) {
+          System.out.println("\nWarning: Specify the --content flag to update a secret's content.");
+          System.out.println("The secret has not been updated with any provided content.");
+        }
+      } catch (IOException e) {
+        throw new AssertionError(String.format("Unexpected error trying to create an InputStreamReader for stdin: %s", e.getMessage()));
```

Comment:
Maybe we should just log & swallow this?  We're only constructing the InputStreamReader for a warning